analysis/GreenSKU-Framework/server_configs/README.md

```diff
@@ -0,0 +1,5 @@
+# Server Configurations
+
+This directory contains the server configurations relevant to our paper. Specifically, we provide a set of baseline server configurations and GreenSKU configurations.
+
+The format of the server configurations is in yaml files. Each yaml file contains straightforward fields that account for the different components within a server and the number of each component. There are also field to specify parameters such as the space and power constraints of the rack. Please see the paper for full details.
```

Comment:
The YAMLs are not that friendly to read. Could you also add the data here as markdown?